src/objects/Sprite.js

```diff
@@ -38,9 +38,9 @@ THREE.Sprite.prototype.raycast = ( function () {
 
 		matrixPosition.setFromMatrixPosition( this.matrixWorld );
 
-		var distance = raycaster.ray.distanceToPoint( matrixPosition );
+		var distanceSq = raycaster.ray.distanceSqToPoint( matrixPosition );
 
-		if ( distance > this.scale.x ) {
+		if ( distanceSq > this.scale.x * this.scale.x ) {
 
```

Comment:
This line in the sprite raycasting code,

    if ( distance > this.scale.x ) {

is a hack. It does not take into consideration the shape of the sprite, or the sprite's transparent areas.

Given that it is a hack, and given that we are now using the squared distance, perhaps we should do this instead:

    if ( distanceSq > this.scale.x * this.scale.y ) {